app/src/main/res/layout/activity_main.xml

```diff
@@ -9,6 +9,16 @@
     android:id="@+id/container"
     tools:context=".ui.lobby.LobbyActivity">
 
+    <ProgressBar xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/progressBar"
+        android:layout_width="wrap_content"
```

Comment:
Please make sure ProgressBar is horizontal and occupies entire screen width, as we talked earlier.